Assets/Scripts/Selling/SellingPrompt.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Written by Ellie McDonald
+// 04/22/21 
+// This class will control when the selling canvas are enabled 
+using UnityEngine;
+
+public class SellingPrompt : MonoBehaviour
+{
+    private Vector3 player, sellingShed;
+    private Canvas sellPotionCanvas, potionInventoryCanvas;
+
+    private bool enablePrompt = false;
+    private AbstractCalculate distance = new ThreeDimensionalCalculateSellingShed();
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        player = GameObject.Find("Male A").transform.position;
+        sellingShed = GameObject.Find("Selling Shed").transform.position;
+        sellPotionCanvas = GameObject.Find("SellPotionCanvas").GetComponent<Canvas>();
+        potionInventoryCanvas = GameObject.Find("PotionInventoryCanvas").GetComponent<Canvas>();
+    }
+
+    void OnGUI()
+    {
+        player = GameObject.Find("Male A").transform.position;
+
+        // this technically shouldn't change positions so idk if this needs to be int the OnGui() or not
+        sellingShed = GameObject.Find("Selling Shed").transform.position;
+
+        enablePrompt = distance.Calculate(player, sellingShed);
+
+        if (enablePrompt && (!potionInventoryCanvas.enabled))
```

Comment:
Couldn't this just be an if else? Trying to make conditionals easier to read and more "adjustable" in the future

Reply:
I did have an if else originally but it wasn't working. I'll try again.

Reply:
Okay. I was thinking something along the lines of:
 if (enablePrompt) ....
else....

Otherwise our conditionals will be super hard to read and modify. Although we really won't be doing anymore work on this project after Wednesday :)